src/vs/workbench/contrib/notebook/browser/contrib/navigation/arrow.ts

```diff
@@ -38,6 +39,7 @@ registerAction2(class FocusNextCellAction extends NotebookCellAction {
 		super({
 			id: NOTEBOOK_FOCUS_NEXT_EDITOR,
 			title: localize('cursorMoveDown', 'Focus Next Cell Editor'),
+			precondition: CONTEXT_ACCESSIBILITY_MODE_ENABLED.negate(),
```

Comment:
Putting it in `precondition` will make it impossible to use this command in accessibility mode, would adding it to keybinding when clause be better?

Reply:
I considered that, but don't we want screen reader users to not even know abt this command as it is confusing for them?